app/src/main/java/ch/epfl/cs311/wanderwave/ui/App.kt

```diff
@@ -41,8 +46,14 @@ fun App(navController: NavHostController) {
 fun AppScaffold(navController: NavHostController) {
   val navActions = NavigationActions(navController)
   var showBottomBar by remember { mutableStateOf(false) }
-
   val currentRouteState by navActions.currentRouteFlow.collectAsStateWithLifecycle()
+  val snackbarHostState = remember { SnackbarHostState() }
+
```

Comment:
I guess it doesn't matter that much for now but Snackbar in middle would be nicer instead of on the bottom which is right on top of the bottom bar.

Reply:
I think in Material 3, Snackbars are intended to be at the bottom for small messages that persist across different screens. For important modal dialogs it's probably better to put them directly in the screens, as the user won't be navigating while they are shown anyway.